### PR TITLE
Admins can now force night shift mode via secrets

### LINKED ISSUE #36103: There is no verb/secrets button to tweak night shift mode
VV is nice, but it's not the be all end all to admin's interface into the round. Advance round state is generally manipulated via the secrets panel and so night shift mode should have buttons there to edit it.

One to force toggle day mode and night mode, and one to toggle automatic switching.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #36103

### CODE REVIEW

**Comment by Iamgoofball** [comment]
We shouldn't be handing out GBP for feature requests. @Cyberboss @KorPhaeron

**Comment by ike709** [comment]
image
Especially when it has the "not a bug" label.

**Comment by kevinz000** [comment]
damnit i got busted

**Comment by 81Denton** [comment]
GBP police WEE WOO

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -164,6 +165,23 @@
...
+			if(!check_rights(R_ADMIN))
+				return
+			var/val = alert(usr, "What do you want to set night shift to? This will override the automatic system until set to automatic again.", "On", "Off", "Automatic")
+			switch(val)
+				if("Automatic")
+					if(CONFIG_GET(flag/enable_night_shifts))
+						SSnightshift.can_fire = TRUE
+						SSnightshift.fire()
+					else
+						SSnightshift.activate_nightshift(TRUE)
```
Comment (SpaceManiac) [inline code/modules/admin/secrets.dm]: is this meant to be `deactivate`? also these will need to be changed to `update_nightshift(active, announce)` (#36082)

**Pull request merged**